bindep.txt

```diff
@@ -1,4 +1,4 @@
 # This is a cross-platform list tracking distribution packages needed by tests;
 # see https://docs.openstack.org/infra/bindep/ for additional information.
 
-rsync [platform:rhel-8 platform:rhel-9]
+rsync [platform:redhat]
```

Comment:
```suggestion
rsync [platform:rpm]
```
Here's an alternative solution suggested by @shanecmd
This makes the assumption that the `rsync` package exists on all other rpm-based distros.
Probably a fair assumption.